wire-runtime-swift/src/main/swift/ProtoCodable/ProtoIntCodable+Implementations.swift

```diff
@@ -104,10 +109,10 @@ extension Int64: ProtoIntCodable, ProtoDefaultedValue {
             self = try Int64(bitPattern: reader.readFixed64())
         case .signed:
             // sint64 fields
-            self = try reader.readVarint64().zigZagDecoded()
+            self = try reader.readVarint().zigZagDecoded()
         case .variable:
             // int64 fields
-            self = try Int64(bitPattern: reader.readVarint64())
+            self = try Int64(truncatingIfNeeded: reader.readVarint())
```

Comment:
TODO: I think this needs to be `bitPattern` when converting from UInt64 to Int64.